tasks/requirements_all_tasks.txt

```diff
@@ -1,5 +1,15 @@
--r kernel_matrix_testing/requirements.txt
 -r libs/requirements-github.txt
 -r libs/requirements-notifications.txt
 -r show_linters_issues/requirements.txt
--r requirements_release_tasks.txt
+
+# From release.py (needed to run inv update-build-links)
+atlassian-python-api==3.41.3
+
+# From test.py (needed to run inv lint-python)
```

Comment:
should we add a cross-reference to https://github.com/DataDog/datadog-agent-buildimages/blob/main/requirements/circleci.txt#L4 and/or https://github.com/DataDog/datadog-agent-buildimages/blob/main/requirements/constraints.txt ?

Reply:
I agree it would be nice if we could rely on a single definition done on the image to ensure consistency